client/extensions/wp-super-cache/miscellaneous.jsx

```diff
@@ -33,7 +33,7 @@ const Miscellaneous = ( {
 	translate,
 } ) => {
 	return (
-		<div>
+		<div className="wp-super-cache__miscellaneous-section">
```

Comment:
We should be able to add a `className` prop directly to the `Notice` component and avoid having to add a class at this level.